rjc_stock/views/stock_move_views.xml

```diff
@@ -7,7 +7,9 @@
         <field name="arch" type="xml">
             <field name="lot_id" position="attributes">
                 <attribute name="options">{'create': True}</attribute>
+                <attribute name="options">{'create_edit': False}</attribute>
```

Comment:
If you comment `<attribute name="options">{'create': True}</attribute>` out, Is that OK ?

You don't need to add `<attribute name="options">{'create_edit': False}</attribute>`